app/services/generaldetails/CandidateDetailsService.scala

```diff
@@ -48,22 +49,27 @@ trait CandidateDetailsService {
     val updatePersonalDetailsFut = pdRepository.update(applicationId, userId, personalDetails,
       sourceStatuses = List(CREATED, IN_PROGRESS), targetStatus = IN_PROGRESS)
     val contactDetailsFut = cdRepository.update(userId, contactDetails)
+    val fastPassDetailsFut = fpdRepository.update(applicationId, candidateDetails.fastPassDetails)
```

Comment:
Do we really need another repository for fastPassDetails, isn't it another part of the personal details? We can save a call to mongo. For ContactDetails it make sense because is a separate collection